product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -595,3 +595,11 @@ bdr.alter_node_option(node_name text,
 -   `node_name` &mdash; Name of the node to be changed.
 -   `config_key` &mdash; Key of the option in the node to be changed.
 -   `config_value` &mdash; New value to be set for the given key.
+
+The node options that can be changed using this function are:
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '100'.
+- `route_fence` &mdash; Whether the node is fenced from routing; when true, the node can't receive connections from PGD Proxy. Default is 'f' (false).
+- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is 't' (true).
+- `route_reads` &mdash; Boolean: Whether read-only connections can be routed to this node. Currently reserved for future use. Default is 't' (true).
+- `route_dsn` &mdash; The dsn that the proxy will use to connect to this node.
```

Comment:
May be we can add word  - `(Optional)`. If not specified it defaults to `bdr.node#node_dsn` 

(`COALESCE(node_config.node_route_dsn, node.node_dsn) AS route_dsn`)

```
bdrdb=# \d+ bdr.node_routing_config_summary
                       View "bdr.node_routing_config_summary"
     Column     |  Type   | Collation | Nullable | Default | Storage  | Description 
----------------+---------+-----------+----------+---------+----------+-------------
 node_name      | name    |           |          |         | plain    | 
 route_priority | integer |           |          |         | plain    | 
 route_fence    | boolean |           |          |         | plain    | 
 route_writes   | boolean |           |          |         | plain    | 
 route_reads    | boolean |           |          |         | plain    | 
 route_dsn      | text    |           |          |         | extended | 
View definition:
 SELECT node.node_name,
    node_config.node_route_priority AS route_priority,
    node_config.node_route_fence AS route_fence,
    node_config.node_route_writes AS route_writes,
    node_config.node_route_reads AS route_reads,
    COALESCE(node_config.node_route_dsn, node.node_dsn) AS route_dsn
   FROM bdr.node
     LEFT JOIN bdr.node_config USING (node_id);
```